irc_bot/irc_bot.py

```diff
@@ -163,6 +163,7 @@ def __init__(self, config):
         self.invite_message = config.get('invite_message')
         self.nickserv_password = config.get('nickserv_password')
         self.use_ssl = config.get('use_ssl', False)
+        self.use_password = config.get('use_password', None)
```

Comment:
Ah! I will try that. This is my first pull request ever. Would you like me to change `self.use_password` to `self.password` as well?